requirements.txt

```diff
@@ -4,6 +4,6 @@ galaxy_importer>=0.4.5,<0.5
 GitPython>=3.1.24,<3.2
 jsonschema>=4.4,<4.7
 packaging>=21.3,<22
-pulpcore>=3.20,<3.25
+pulpcore>=3.21.dev,<3.25
```

Comment:
I can see how this is required, but I guess we'll just have to remember to change this at release time?

Reply:
I meant more to remember to change s/3.21.dev/3.21/ at release time, but I do think this is better than not changing it now.